frontend/src/components/commons/PopupBox.jsx

```diff
@@ -0,0 +1,150 @@
+/*
```

Comment:
It's best practice for imports should always be the first thing in a file.

Comments explaining the component should appear just before the component definition on line 50.

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/126/commits/062f7c2130db3071541c35f4e811abe81624f178